.storybook/main.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
     "core": {
         "builder": "@storybook/builder-webpack5"
     },
```

Comment:
MSW in StoryBook build and during the dev run needs service worker available on root level